src/index.ts

```diff
@@ -1 +1,9 @@
 export * from './components';
+
+const createElement = document.querySelector('manifold-subscription-create');
+
+// eslint-disable-next-line no-unused-expressions
+createElement?.addEventListener<any>('onSuccess', (e: any) => {
+  console.log(e);
+  alert(e.detail.id);
+});
```

Comment:
This is just to test out the event. It doesn't seem to be working. I've tried `success` `onSuccess`